README.md

```diff
@@ -49,7 +49,25 @@ sed -i "s/<REPLACE ME>/GITHUB_NICK/g" boxes.d/99-local.yaml
 vagrant up centos7-devel
 ```
 
-You can find more thorough guides in the [docs folder](https://github.com/theforeman/forklift/tree/master/docs).
+In case using vagrant is not desired, ansible playbooks and roles from this repo can be used separately. This is useful if existing host should be used for the installation, e.g. a beaker machine. In order to deploy devel environment on host test.example.com, following needs to be done:
```

Comment:
s/existing host/an existing host/
s/deploy devel/deploy the devel/
s/following needs/the following needs/

Beaker machines might not mean a lot to upstream users so I'd leave it out.